src/vimeo/players/VimeoDesktopArticleVideoPlayer.tsx

```diff
@@ -2,66 +2,55 @@ import React, { useRef, useState } from 'react';
 import { VimeoArticleVideoPlayerProps } from 'vimeo/types';
 import useOnScreen from 'utils/useOnScreen';
 import PlayerWrapper from 'vimeo/players/shared/PlayerWrapper';
-import WDSVariables from '@fandom-frontend/design-system/dist/variables.json';
 import CloseButton from 'vimeo/players/shared/CloseButton';
 import VimeoPlayerWrapper from 'vimeo/players/shared/VimeoPlayerWrapper';
-import styled, { css, keyframes } from 'styled-components';
 
-const VimeoDesktopArticleVideoTopPlaceholder = styled.div`
-	position: absolute;
-	width: 100%;
-	padding-top: 56.25%;
-	top: 0;
-	left: 0;
-	bottom: 0;
-	right: 0;
-	z-index: ${Number(WDSVariables.z2)};
-`;
+import clsx from 'clsx';
 
-const moveDownAnimation = (right: number, width: number) => keyframes`
-	from {
-		right: ${right}px;
-		bottom: 100%;
-		width: ${width}px;
-	}
-
-	to {
-		right: 18px;
-		bottom: 45px;
-		width: 300px;
-	}
-`;
+import styles from './VimeoDesktopArticleVideoPlayer.module.css';
 
 interface VimeoDesktopArticleVideoWrapperProps {
 	isScrollPlayer: boolean;
 	right?: number;
 	width?: number;
 }
 
-const DesktopArticleVideoWrapper = styled.div<VimeoDesktopArticleVideoWrapperProps>`
-	height: max-content;
-	${(props) =>
-		props.isScrollPlayer
-			? css`
-					z-index: ${Number(WDSVariables.z2) + 2};
-					position: fixed;
-					animation: ${moveDownAnimation(props.right, props.width)} 0.4s normal forwards;
-			  `
-			: css`
-					position: absolute;
-					bottom: 0;
-					right: 0;
-					top: 0;
-					left: 0;
-					transform: translateZ(0);
-					-webkit-transform: translateZ(0);
-			  `}
-`;
+const DesktopArticleVideoWrapper: React.FC<VimeoDesktopArticleVideoWrapperProps> = ({
+	isScrollPlayer,
+	right,
+	width,
+}) => {
+	return (
+		<>
+			<style>
+				{`
+					@keyframes moveDownAnimation {
+						from {
+							right: ${right}px;
+							bottom: 100%;
+							width: ${width}px;
+						}
+
+						to {
+							right: 18px;
+							bottom: 45px;
+							width: 300px;
+						}
+					}
```

Comment:
I wonder if there is a way to make it more readable (less indentation). What about moving it to `./VimeoDesktopArticleVideoPlayer.module.css`, would it work? 🤔

Reply:
It seems like the formatter here is a problem. This looks completely different on my end 🙈